documentation/docs/pages/operators/nodes/preliminary-steps/vps-setup/advanced.mdx

```diff
@@ -510,11 +510,46 @@ Your KVM installation is now ready to deploy and manage VMs.
 
 </Steps>
 
-### Setting Up Virtual Machines
+## Setting Up Virtual Machines
 
-After finishing the [installation of KVM](#installing-kvm-on-a-server-with-ubuntu-2204), we can move to the virtualisation configuration.
+**After finishing the [installation of KVM](#installing-kvm-on-a-server-with-ubuntu-2204), we can move to the virtualisation configuration.**
 
-> **The steps below will guide you through a setup of one VM, therefore you will have to repeat this process for each VM**. That also means that you have to be mindful of space and memory allocation.
+> **This tutorial will guide you through a setup of one VM, therefore you will have to repeat this process for each VM**. That also means that you have to be mindful of space and memory allocation.
```

Comment:
It might be helpful here to say something re: how to distribute disc space / mem to each VM - whether it is a good idea to do it equally, to leave a certain % free for root, etc?